xapian-bindings/java/SmokeTest.java

```diff
@@ -121,8 +121,20 @@ public static void main(String[] args) throws Exception {
 		System.exit(1);
 	    }
 	    Enquire enq = new Enquire(db);
-	    enq.setQuery(new Query(Query.OP_OR, "there", "is"));
+	    enq.setQuery(new Query("dog"));
 	    MSet mset = enq.getMSet(0, 10);
+	    // MSet size is 0
+	    if (mset.size() == 0) {
+	    for (MSetIterator.MatchDescriptor descriptor : mset) {
+	    System.err.println("Empty iterator entered invalid section (from MSet java iterator)");
```

Comment:
I kept tabs at those lines as you have mentioned but when I run `git diff origin/master | ./xapian-maintainer-tools/xapian-check-patch`, I kept getting the message "added/changed line has space before tab" so I just removed those tabs. 
The rest of the file also has indentation mistakes within if blocks, I wonder why those aren't showing up.